docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx

```diff
@@ -0,0 +1,96 @@
+---
+slug: /quickstart/883939/containers
+hide_table_of_contents: true
+title: "Chain container functions"
+---
+import PartialExperimentalDocs from '../partials/_experimental.mdx';
+
+# Quickstart
+
+<PartialExperimentalDocs />
+
+## Chain container functions
+
+Just as you can chain `Directory` function calls, you can also chain `Container` function calls. As an example, the Wolfi container builder module exposes a number of functions:
```

Comment:
FIXME, I haven't reviewed this page, but the idea is to make it symmetrical to the "building binaries" page.

Reply:
Modified structure